cluster/examples/workloads/kubernetes/wordpress/gcp/environment.yaml

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
Is this renamed `environment.yaml` just to be more consistent with its functionality?